op-guide/ansible-deployment.md

```diff
@@ -5,104 +5,282 @@ category: operations
+
+Make sure you have logged in to the Control Machine using the `tidb` user account.
+
+It is required to use `pip` to install Ansible and its dependencies, otherwise a compatibility issue occurs. Currently, the TiDB 2.0 GA version and the master version are compatible with Ansible 2.4 and Ansible 2.5.
 
 1. Install Ansible and the dependencies on the Control Machine:
 
     ```bash
-    sudo yum -y install epel-release
-    sudo yum -y install python-pip curl
-    cd tidb-ansible
-    sudo pip install -r ./requirements.txt
+    $ cd /home/tidb/tidb-ansible
+    $ sudo pip install -r ./requirements.txt
     ```
 
-    Ansible and related dependencies are in the `tidb-ansible/requirements.txt` file.
+    Ansible and the related dependencies are in the `tidb-ansible/requirements.txt` file.
 
 2. View the version of Ansible:
 
     ```bash
-    ansible --version
+    $ ansible --version
+    ansible 2.5.0
+    ```
+
+## Step 5: Configure the SSH mutual trust and sudo rules on the Control Machine
+
+Make sure you have logged in to the Control Machine using the `tidb` user account.
+
+1. Add the IPs of your target machines to the `[servers]` section of the `hosts.ini` file.
+
+    ```bash
+    $ cd /home/tidb/tidb-ansible
+    $ vi hosts.ini
+    [servers]
+    192.168.0.2
+    192.168.0.3
+    192.168.0.4
+    192.168.0.5
+    192.168.0.6
+    192.168.0.7
+    192.168.0.8
+    192.168.0.10
+
+    [all:vars]
+    username = tidb
+    ntp_server = pool.ntp.org
+    ```
+
+2. Run the following command and input the `root` user account password of your target machines.
+
+    ```bash
+    $ ansible-playbook -i hosts.ini create_users.yml -k
+    ```
+
+    This step creates the `tidb` user account on the target machines, configures the sudo rules and the SSH mutual trust between the Control Machine and the target machines.
+
+## Step 6: Install the NTP service on the target machines
+
+> **Note:** If the time and time zone of all your target machines are same, the NTP service is on and is normally synchronizing time, you can ignore this step. See [How to check whether the NTP service is normal](#how-to-check-whether-the-ntp-service-is-normal).
+
+Make sure you have logged in to the Control Machine using the `tidb` user account, run the following command, and input the `root` password of your target machines as prompted:
+
+```bash
+$ cd /home/tidb/tidb-ansible
+$ ansible-playbook -i hosts.ini deploy_ntp.yml -k
+```
+
+The NTP service is installed and started using the software the comes with the system on the target machines. The default NTP server list in the installation package is used. The related `server` parameter is in the `/etc/ntp.conf` configuration file.
+
+To make the NTP service start synchronizing as soon as possible, the system executes `ntpdate` one time on `ntp_server` in the `hosts.ini` file. The default server is `pool.ntp.org`, and you can also replace it with your NTP server.
```

Comment:
the system executes `ntpdate` sets the local date and time by polling `ntp_server` in the `hosts.ini` file.